### ISSUE TITLE
Double /trunk when templating SVN repos

### ISSUE THREAD

**Issue by LGJLI** [scm-manager/scm-repository-template-plugin#13]
## Issue description

Trunk folder is copied when using SVN repo as template. 

### Bug Report

When using SVN repos as templates, the `template.yml` file is placed in the root next to the `/trunk` folder. Giving the following folder structure:
```
template.yml
trunk
trunk/README.md
```

If the template is used as base for a new repo, the trunk is copied which yields the following folder structure
```
trunk
trunk/README.md
trunk/trunk/
trunk/trunk/README.md
```

Maybe I am just not understanding how I should setup the template.yml file ? 
I tried different combinations of adding `trunk` to filters, but nothing worked. 

#### Describe how to reproduce the bug
1. Create new blank SVN repo.
2. Set the repo to be a template
3. create new SVN repo based on template

#### Describe your SCM-Manager setup
Vanilla installation 3.03 on Windows server 2019. 
Adoptium JDK 21.0.2+13
Repository template 1.2.0

#### Which operating system, browser and versions do you use?
Windows 10 22H2, Firefox 122.0

#### Further information like screenshots

![image](https://github.com/scm-manager/scm-repository-template-plugin/assets/93387775/e068950f-c819-4f88-8368-826f5c68b53b)

**Comment by pfeuffer** [comment]
Hey @LGJLI , this definitely sound like a bug. Thanks for the report, we will take a look at this.

**Comment by christophloose** [comment]
HI @LGJLI , we will have a look at it probably next week.

**Comment by christophloose** [comment]
Hi @LGJLI could you please share your template.yml file? Thanks a lot!

**Comment by LGJLI** [comment]
Can't upload the file due to github restrictions, but the contents are the default when setting repo as template. 
```yml
engine: mustache
encoding: UTF-8
files:
  - path: ""
    filtered: false
```

**Comment by fscholdei** [comment]
The fix should be available with the next release of the repository template plugin probably in mid-May.

**Comment by fscholdei** [comment]
Fixed with [version 1.2.2](https://scm-manager.org/plugins/scm-repository-template-plugin/releases/).
